target_chains/ethereum/contracts/contracts/pulse/Pulse.sol

```diff
@@ -13,16 +13,24 @@ abstract contract Pulse is IPulse, PulseState {
         address admin,
```

Comment:
just to be clear, here the default provider is just set and should be initialized in a separate tx? (maybe make a comment on it)

Reply:
yup, will do